esp32/sdkconfig.h

```diff
@@ -36,7 +44,6 @@
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK 0
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK_CPU1 0
 
-#define CONFIG_FREERTOS_UNICORE 1
```

Comment:
> some (new) functions are hard-coded to use 2 cores

Can you give examples of these?  I'd be surprised if the IDF changed so much that it no longer works with single core.

> I was just in "make it work" mode.

The esp32 port is still beta so as long as things work then it's ok.  I assume you checked that wifi and your BT stuff works.